extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/telemetry/WebsocketTelemetryProvider.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.websockets.next.runtime.telemetry;
+
+import java.util.function.Function;
+
+import io.quarkus.websockets.next.runtime.WebSocketConnectionBase;
+import io.quarkus.websockets.next.runtime.WebSocketEndpoint;
+
+public final class WebsocketTelemetryProvider {
```

Comment:
```suggestion
public final class WebSocketTelemetryProvider {
```
Just for consistency...